### ISSUE TITLE
Regression of Function.constructor

### ISSUE THREAD

**Issue by edgemaster** [resource-reasoning/JavaScriptVerification#6]
test/language/function-code/S10.2.1_A4_T2.js :
```javascript
function f1(){
  var x;

  return x;

  function x(){
    return 7;
  }
}

__$ERROR__ = f1().constructor.prototype === Function.prototype;
```
Should return true, it does in the version in resource-reasoning/JS_Symbolic_Debugger, the current version in JavaScriptVerification has regressed to false.

`f1().constructor.prototype` is `#lfp` in JS_S_D, but `#lap` in JSV.

**Comment by edgemaster** [comment]
I've just run `git bisect` over the repo, apparently 51a66d895b25cac7c816f275aa330cf6e7b3087a caused the breakage.

**Comment by edgemaster** [comment]
Ah, here we go, a copy paste error: https://github.com/resource-reasoning/JavaScriptVerification/commit/51a66d895b25cac7c816f275aa330cf6e7b3087a#diff-fdedc4664182a8a866c0cad5b370097dR707

**Comment by da319** [comment]
Yes, my fault. I will fix it.

Daiva

**Comment by edgemaster** [comment]
Fixed in 37a3b1c

**Comment by da319** [comment]
:)
